pandas/core/aggregation.py

```diff
@@ -198,6 +293,79 @@ def maybe_mangle_lambdas(agg_spec: Any) -> Any:
     return mangled_aggspec
 
 
+def _relabel_result(
```

Comment:
can you make this public (no _) as we are calling from a related module (its not exposed to users at all, just easier to grok)